### PR TITLE
feat: add stop gradient calls for certain cases for beta/nu in Moffat and Spergel plus docs

### PR DESCRIPTION
**Pull request by beckermr**
This is more for completeness than anything else.

### CODE REVIEW

File: tests/jax/test_deriv_gsobject.py [hunk 1]
```diff
@@ -44,3 +44,29 @@ def _run(val_):
...
+    eps = 1e-5
+
+    def _run(val_):
+        return jnp.max(
+            jgs.Spergel(
+                val,
+                scale_radius=2.0,
+                gsparams=jgs.GSParams(minimum_fft_size=8, maximum_fft_size=8),
+            )
+            .drawImage(nx=5, ny=5, scale=0.2, method="fft")
```
Comment (beckermr) [inline tests/jax/test_deriv_gsobject.py]:
```suggestion
            .drawImage(nx=5, ny=5, scale=0.2)
```
